aiida/cmdline/commands/cmd_export.py

```diff
@@ -141,19 +142,20 @@ def create(
 
 @verdi_export.command('migrate')
 @arguments.INPUT_FILE()
-@arguments.OUTPUT_FILE()
+@arguments.OUTPUT_FILE(required=False)
 @options.ARCHIVE_FORMAT()
 @options.FORCE(help='overwrite output file if it already exists')
+@click.option('-i', '--in-place', is_flag=True, help='Migrate archive in placePrint the data contents and exit.')
 @options.SILENT()
 @click.option(
     '-v',
     '--version',
     type=click.STRING,
     required=False,
     metavar='VERSION',
-    help='Specify an exact archive version to migrate to. By default the most recent version is taken.'
+    help='Archive format version to migrate to (default: latest version).'
```

Comment:
Maybe better to use an actual default? As in, import the latest version from `aiida.tools.importexport`, set that as the default and set `show_default=True`.